compiler/rustc_infer/src/infer/mod.rs

```diff
@@ -678,8 +678,6 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
 
     pub fn unsolved_variables(&self) -> Vec<Ty<'tcx>> {
         let mut inner = self.inner.borrow_mut();
-        // FIXME(const_generics): should there be an equivalent function for const variables?
```

Comment:
I also took the opportunity to remove this `FIXME`. This doesn't require anything for const variables, because there's no notion of fallback for const generics.